appveyor.yml

```diff
@@ -0,0 +1,30 @@
+version: 1.0.{build}
+
+platform: x64
+shallow_clone: true # Because the git clone takes ages due to its size
+
+# Cf. https://stefanscherer.github.io/setup-windows-docker-ci-appveyor/
+# Visual Studio 2017 allows to run Windows's Containers 
+image:
+- Visual Studio 2017
+
+# Show Windows and docker versions
+install:
+- ps: Write-Host Server version $(gp 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion').BuildLabEx
+- docker version
+
+build:
+  verbosity: minimal
+
+build_script:
+- ps: docker build -t traefik:windows ./windows/
+
+test_script:
+- ps: >-
```

Comment:
When you use `- ps: |` you can omit the empty lines.

```yaml
- ps: |
    docker run ...
    docker ps
    Invoke-WebRequest ...
```

Reply:
👍 fixed